ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/location/keyword.spec.js

```diff
@@ -33,51 +33,51 @@ const { shallow } = Enzyme
 describe('<Keyword />', () => {
   before(() => {
     mockProperties()
-    Keyword = require('./keyword')
+    Keyword = require('./keyword').default
   })
   after(() => {
     unmockProperties()
   })
-  it('should fetch features on select', (done) => {
-    const coordinates = [1, 2, 3]
-    const fetch = async (url) => {
-      expect(url).to.equal('./internal/geofeature?id=0')
-      return {
-        async json() {
-          return {
-            geometry: {
-              type: 'Polygon',
-              coordinates: [coordinates],
-            },
-          }
-        },
-      }
-    }
-    const setState = ({ polygon }) => {
-      expect(wrapper.state('loading')).to.equal(false)
-      expect(polygon).to.equal(coordinates)
-      done()
-    }
-    const wrapper = shallow(<Keyword fetch={fetch} setState={setState} />)
-    wrapper.find('AutoComplete').prop('onChange')({
-      id: '0',
-      name: 'test',
-    })
-    expect(wrapper.state('loading')).to.equal(true)
-  })
-  it('should inform user on failure to fetch geofeature', (done) => {
-    const onError = () => {
-      wrapper.update()
-      expect(wrapper.text()).include('Geo feature endpoint unavailable')
-      done()
-    }
-    const fetch = async () => {
-      throw new Error('unavailable')
-    }
-    const wrapper = shallow(<Keyword fetch={fetch} onError={onError} />)
-    wrapper.find('AutoComplete').prop('onChange')({
-      id: '0',
-      name: 'test',
-    })
-  })
+  // it('should fetch features on select', (done) => {
```

Comment:
✏️ Should we just delete all of these tests if we don't intend to replace them now?

Reply:
I'm not sure to be honest, I left them commented out as a guide for what we should try and fix.  They've been not running for so long that I'm not entirely sure how relevant they are, but they probably have good ideas of what we should be testing in the components (the components have just changed too much). 

I originally had wanted to add some tests to ensure the filter builder didn't regress, but ran into issues just trying to render it.  Didn't have time to chase them down, but thought I'd include the change to make it possible to turn tests back on.

Reply:
We could always check an earlier commit to reference these, but it's not a big deal me 🤷. I'm sure we'll revisit these as we increase our testing efforts here soon.